core/templates/dev/head/pages/exploration-editor-page/translation-tab/modal-templates/add-audio-translation-modal.template.html

```diff
@@ -32,7 +32,7 @@ <h3>Add Audio Translation</h3>
 
 <div class="modal-footer">
   <button class="btn btn-secondary protractor-test-close-audio-upload-modal" ng-click="cancel()">Cancel</button>
-  <button class="btn btn-success protractor-test-save-uploaded-audio-button" ng-click="save()" ng-disabled="!isAudioTranslationValid() || saveInProgress">
+  <button class="btn btn-success  protractor-test-save-uploaded-audio" ng-click="save()" ng-disabled="!isAudioTranslationValid() || saveInProgress">
```

Comment:
why these changes? I don't think you need the double space, and I think it is ok to suffix the class name with the type of object (button).

Reply:
Resolved in f9c16e2d3